test/basic.js

```diff
@@ -128,6 +130,31 @@ t.test('--harmony', function (t) {
   })
 })
 
+t.test('node exe with different name', function(t) {
+  var fp = path.join(__dirname, 'fixtures', 'exething')
+  fs.createReadStream(process.execPath)
```

Comment:
Sorry, that should have been clearer. 

Example:

```bash
$ ln -s /usr/local/bin/node ./exething
$ ./exething -pe process.execPath
/usr/local/bin/node
```

So doing it that way didn't actually the change being made